bower.json

```diff
@@ -7,7 +7,7 @@
 		"external",
 		"tests"
 	],
-+       "license": "MIT",
+        "license": "MIT",
```

Comment:
This doesn't respect the code style of the whole file. A single tab should be used.